tokio/src/task/stack.rs

```diff
@@ -28,7 +28,7 @@ impl<T: 'static> TransferStack<T> {
 
             // At this point, the queue_next field may also be used to track
             // whether or not the task must drop the join waker.
-            debug_assert_eq!(0, next & 1);
+            debug_assert_eq!(0, next & !1);
```

Comment:
was the previous assertion incorrect?

Reply:
yes, but debug assertions are not enabled when running loom in release mode...